pkg/image/builder/image_builder.go

```diff
@@ -80,85 +81,196 @@ func (b *ImageBuilder) Run(ctx context.Context, targetRef string, layoutPath lay
 	if err != nil {
 		return err
 	}
-	idxManifest, err := idx.IndexManifest()
+	// make a copy of the original manifest for later
+	originalIdxManifest, err := idx.IndexManifest()
 	if err != nil {
 		return err
 	}
+	originalIdxManifest = originalIdxManifest.DeepCopy()
```

Comment:
Hi @jchunkins 
Just a nit pick, and for me to be sure there's nothing that we are overwriting by mistake: Is this the equivalent to: 
```suggestion
	originalIdxManifest, err := idx.IndexManifest().DeepCopy()
	if err != nil {
		return err
	}

```

Reply:
Yes, this would be equivalent (at least from a pseudo code standpoint). I can't do that directly due to the two valued return (i.e. the error value).